scripts/library.sh

```diff
@@ -411,12 +411,29 @@ function start_knative_serving() {
   header "Starting Knative Serving"
   subheader "Installing Knative Serving"
   echo "Installing Serving CRDs from $1"
-  kubectl apply --selector knative.dev/crd-install=true -f "$1"
+  kubectl apply --selector knative.dev/crd-install=true -f "$1" || return 1
```

Comment:
Hmmm... I understand your change here, but I would prefer not touching `start_knative_serving` in this PR unless we had evidence that this is a bug. Otherwise I'm concerned that you might inadvertently cause issues in the next test-infra update.

Reply:
Removed.